src/Steeltoe.Initializr.WebApp/ClientApp/src/components/BottomLinks.js

```diff
@@ -6,8 +6,7 @@ export class BottomLinks extends Component {
     return (
 
     <div className="left">
-        <div className="footer">© 2013-2019 Pivotal Software<br/>start.steeltoe.io is powered by<br/>
-            <a href="https://run.pivotal.io/" tabIndex="999">Pivotal Web Services</a><br/>v 0.6.0</div>
+        <div className="footer">© 2019-2020 VMware, Inc. All Rights Reserved<br/>v 0.6.2</div>
```

Comment:
Minor - but VMware, Inc. should be a link to https://vmware.com

Reply:
added link